enhancements/olm/platform-operators.md

```diff
@@ -0,0 +1,609 @@
+these non-essential operators are still included in the lifecycle of cluster installations and upgrades. The Platform Operator (PO) concept will allow cluster administrators to choose which of these non-control-plane-essential operators to include in the cluster lifecycle.
+
+Secondly, some portion of the layered product operators in the OpenShift catalogs are conducive with inclusion in the cluster lifecycle (e.g. cert-manager, service binding operator, and the local storage operator). The Platform Operator concept will open the door for more of OLM’s optional operators to be included in (and impact) the cluster lifecycle. This will allow customers to further tailor
+their cluster installs and upgrades to their business needs.
+
+Lastly, the introduction of the PO concept will enable independent components to ship on their own release cadence, and independent of the OpenShift [payload lifecycle](#payload-lifecycle). Future iterations of the "Composable OpenShift" initiative will lead to the eventual delegating of optional, CVO-based components to OLM through this PO mechanism.
+
+### User Stories
+
+- As a cluster admin, I want to be able to install OLM-based Operators at cluster creation, and participate in the cluster's lifecycle.
+- As a cluster admin that installed a PO that's no longer needed, I want to be able to delete a PO without having to uninstall the cluster.
+- As a cluster admin, I want to be able to approve new versions of an individual PO package.
+- As a team that has a direct dependency on an OLM-based Operator, I want to be able to tailor my installation to meet my use case.
+- As an operator author, I want admins to be able to get my operator installed by default and automatically upgraded as the cluster upgrades so that I can provide "core" cluster functionality that looks cohesive with the rest of the cluster/payload.
+
+### Goals
+
+- Admins can select and configure POs before cluster creation.
+- Admins can install and delete a PO after cluster creation.
+- Ensure that OLM-based operators can be installed during day 0 operations.
+- Ensure that any failed PO installations result in a failed cluster installation.
+- Ensure that only AllNamespace OLM-based operators can be installed.
+- Respect any Kubernetes/OCP cluster version constraints while selecting candidate bundles.
+- Respect the "operatorframework.io/suggested-namespace" CSV annotation value.
+
+### Non-Goals
+
+- Prohibit multiple PO packages from being installed on the cluster during phase 0.
+- Provide a first-class UX for configuring POs via the install-config API during phase 0.
```

Comment:
This implies to me that we are not GA-ing this at first, can we make sure to have a goal or non goal that states we are introducing this as a tech preview/not releasing as GA however you decide you prefer to write it up